src/classes/STG_PanelRDHealthCheck_TEST.cls

```diff
@@ -37,6 +37,7 @@
 private class STG_PanelRDHealthCheck_TEST {
 
     private static final String BAD_PICKLIST_VALUE = 'Non Existent Value';
+    private static final String ERROR_MESSAGE = 'Error Message {0} {1}';
```

Comment:
Let's revert Installment Period tests as they were before and use the following at the beginning of each test that is not applicable when RD2 metadata is deployed.
```
        if (isRD2MetadataDeployed()) {
            return;
        }
```